tests/integration/unit_test/test_unit_test_dotnet7.py

```diff
@@ -0,0 +1,11 @@
+# dotnet unit tests are executed from test directories, so code_directories point to test directories
+from tests.integration.unit_test.unit_test_base import UnitTestBase
+
+
+class UnitTest_dotnet6_cookiecutter_aws_sam_hello_dotnet(UnitTestBase.DotNetCoreUnitTestBase):
+    directory = "dotnet7-aot/hello"
+    code_directories = ["test/HelloWorld.Test"]
+
+class UnitTest_dotnet6_cookiecutter_aws_sam_hello_dotnet_pt(UnitTestBase.DotNetCoreUnitTestBase):
+    directory = "dotnet7-aot/hello-pt"
```

Comment:
Is this PR missing the `dotnet7-aot/hello-pt` directory? Also can we change the class name for the tests to be related to the template names, i.e `UnitTest_dotnet7_aot_cookiecutter_hello_pt`

Reply:
removing as that's not supported yet, perhaps @sliedig thought we could've tackled that before the release: https://github.com/awslabs/aws-lambda-powertools-dotnet/issues/212

Reply:
removed